toolkit/dump_tasks.php

```diff
@@ -62,7 +63,10 @@
 		echo "+--------------------------------+----------------------------------------------------+\n";
 		foreach ($aResult['objects'] as $aValues) {
 			$aCurrentTaskDefinition = $aValues['fields'];
-			echo sprintf("| %-30.30s | %-50.50s |\n", $aCurrentTaskDefinition['name'], $aCurrentTaskDefinition['description']);
+			echo sprintf("| %-30.30s | %-50.50s |\n",
+				$aValues['key'].':'.$aCurrentTaskDefinition['name'],
```

Comment:
Why is the Name value changed? Does this new notation needs to be used as value for the `--task_name` parameter?

Reply:
id is added as a suffix, to clear the situation when having multiple datasource with very long names
task_name param value should still be the sole datasource name. Ideally I would add a task_key param to specify the source id instead of name, but maybe in a next pr.

Reply:
@Molkobain indeed, that’s what I was referring to.

Reply:
Are you referring to Molkobain's comment below about moving the id to a new column ? 
If yes, do you agree on the solution ?

Reply:
Yes, it doesn't make sense to change the name of the datasource being displayed if it can't be used like that. A separate column is indeed better.